### ISSUE TITLE
Player Movement

### ISSUE THREAD

**Issue by FelixMo42** [FelixMo42/hex_game#10]
I think it makes sense to move using the WASD keys. W is to the left of S, so W should mean moving to the up-left hex and S should mean moving to the down-right hex. Agree? disagree? (written by ben)

**Comment by FelixMo42** [comment]
If we're using arrow keys to move, then I would vote we rotate the hexs so that it has a flat top. That way we could use a full "qweasd" rather than "wasd".

```
   w
q /‾‾\ e
a \__/ d
   s
```

**Comment by bentref** [comment]
I'm fine with either, qweasd could work well too, but I don't think wasd is too unnatural given the placement of the keys on the keyboard. We could also look into mouse-based movement but that just seems a little less pro-gamer

**Comment by FelixMo42** [comment]
I think it depends on how fast past we want the game. Mouse seems better for slower games, where as keyboard is better for faster games. I personally prefer slower games, so I think I would prefer mouse, but I'm fine with both.

I agree that qweasd is awkward, but If were using the keyboard I think we do need a more complete binding than wasd. I dont know what thought. Maybe awesdf, idk.

**Comment by bentref** [comment]
Yes I have no intention of incorporating fast real-time gameplay, so in that respect mouse-based makes sense. Also I like what you've done so far with the mouse movement so I'm ready to jump on board with a mouse-based game

**Comment by FelixMo42** [comment]
Sounds like a plan to me!
